core/include/moveit/task_constructor/stages/generate_random_pose.h

```diff
@@ -0,0 +1,120 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2020, PickNik Inc
```

Comment:
I guess we've gone ahead in time a "little bit"... haha

```suggestion
 *  Copyright (c) 2023, PickNik Inc